pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
```

Comment:
```suggestion
More information <>.

```

Leave this open and we will decide on a link

Reply:
We tried to introduce 
> More information: N/A.

for pages which didn't have a link a few months ago. But it failed, since tldl-lint wanted to have `< ... >` around the link. Well, then let's introduce:

```suggestion
More information: <N/A>.
```

Reply:
No, I am not suggesting no link. I am just leaving it here to be discussed and the gap to be filled.

https://www.unix.com/man-page/osx/1/nettop/ 
https://apps.apple.com/us/app/nettop/id1212061421?mt=12

Are these any good?

Reply:
Actually the second one looks pretty good. It doesn't give you information on the link, but after you downloaded it you have access to the man page

Reply:
This `nettop` is an OSX builtin command, and preinstalled in system. OSX  user can use  `man nettop` directly.

I think the second link is another different application, but have the same name `nettop` 😂

Reply:
We should just use the unix link

Reply:
Compare to `man nettop`, the unix link seems to be outdated or incomplete , this is the result of `man nettop`  in osx: https://pastebin.com/vqJc1et3

Reply:
We already have the `--help` example to get more information, so I think we could just use `<N/A>`. An outdated man page is worse than none at all.

Reply:
Yes, I agree, An outdated man page is a trap :)

Reply:
Then, could you please apply my suggestion above? I'd be interested if `tldr-lint` complains about it.

Reply:
OK, I have applied your suggestion just now. Leave the next work to `tldr-lint`.